src/fonts/OTF/TeX/eubase.mf

```diff
@@ -1,419 +0,0 @@
-%% @metafontfile{
```

Comment:
I don't think we ever used these files for anything.  Thanks for removing these.

Reply:
@kevinbarabash They were used to trace Euler fonts, as they are not included in the LaTeX distribution.